chemist-aws/src/main/scala/AwsDiscovery.scala

```diff
@@ -157,54 +166,51 @@ class AwsDiscovery(
    * @see funnel.chemist.AwsDiscovery.lookupOne
    */
   protected def lookupMany(ids: Seq[String]): Task[Seq[AwsInstance]] = {
-    def lookInCache: (Seq[String],Seq[AwsInstance]) =
-      ids.map(id => id -> cache.get(id)
-        ).foldLeft[(Seq[String],Seq[AwsInstance])]((Seq.empty, Seq.empty)){ (a,b) =>
+    def lookInCache: (Seq[String], Seq[AwsInstance]) =
+      ids.map(
+        id => id -> cache.get(id)
```

Comment:
can you undo this formating change (and the one below with the fold args)? It's not idiomatic to move these to the next lines